deployment/.surrealdb/aws/deploy-budget.sh

```
@@ -0,0 +1,152 @@
+#!/bin/bash
+set -e
+
+echo "=========================================="
+echo "SurrealDB Budget Deployment on AWS EKS"
+echo "Estimated cost: ~$80-90/month"
```

Shellcheck (0.11.0)

[error] Braces are required for positionals over 9, e.g. ${10}.

(SC1037)